integrations/model-training/sagemaker/log_completed_sagemaker_runs/README.md

```diff
@@ -0,0 +1,91 @@
+# Logging Sagemaker runs to Comet
+
+Comet supports transferring data from Sagemaker runs to Comet's Experiment Management tool. This approach requires no changes
+to existing Sagemake code. It simply transfers data from completed Sagemaker runs to Comet.
+
+There are three ways to log data from a completed Sagemaker Run to Comet
```

Comment:
I think Sagemaker uses jobs instead of runs in general and in this case we are specifically targeting training job. Processing jobs don't have metrics so I'm pretty sure it will always fails. Might be worth renaming the folder also

```suggestion
There are three ways to log data from a completed Sagemaker Training Job to Comet
```